projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.ts

```diff
@@ -32,6 +34,10 @@ export class DataLayerBrowserComponent implements OnInit, AfterViewInit {
       layer.category?.toLowerCase().includes(filter) ||
       layer.datasets.some((dataset) => dataset.label?.toLowerCase().includes(filter));
     this.filterControl.valueChanges.subscribe((value) => (this.dataSource.filter = value?.trim().toLowerCase() ?? ''));
+
+    this.layerManager.loader$.subscribe((loading:boolean) => {
+       this.isProgress = loading;
+    })
```

Comment:
this seems unnecessary. can we just use async pipe?

Reply:
Yes, I am use async pipe instead of this.